qiskit_experiments/curve_analysis/standard_analysis/decay.py

```diff
@@ -76,24 +76,29 @@ def _generate_fit_guesses(
             AnalysisError: When the y data is likely constant.
         """
         curve_data = self._data()
+        return self._initial_guess(user_opt, curve_data.x, curve_data.y)
 
-        user_opt.p0.set_if_empty(base=curve.guess.min_height(curve_data.y)[0])
+    @staticmethod
+    def _initial_guess(
```

Comment:
This separation only makes sense for IRB. This should be implemented in RB analysis.